src/hotspot/share/opto/parse1.cpp

```diff
@@ -2174,16 +2173,50 @@ PhiNode *Parse::ensure_memory_phi(int idx, bool nocreate) {
   return phi;
 }
 
+// Passive Materialization
+// ------------------------
+// C2 has to materialize a virtual object at the merging point because the object has been materialized in
+// any other predecessor. For instance:
+//
+// obj(virtual)     obj(materialized)
+//              \      /
+// Reion(,c1,c2) |    |
```

Comment:
remove comma before `,c1`

`Reion` -> `Region`

Reply:
the comma is there because Region has a fixed form. 
the control of a Region usually is itself. the real predecessors start off 1 rather than 0.